src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -129,9 +129,24 @@ export type AnalyticsTreeCreationNextstrain = {
   // Type of tree being created
   tree_type: string;
   // Location of samples used for tree creation. A null value indicates that
-  // the users group info was not successfully fetched before they created
+  // the user's group info was not successfully fetched before they created
   // a tree. This generally shouldn't happen.
   location_id: number | null;
+  // Location of the user's group. This is provided for comparison with the
+  // location_id above. A null value indicates that the user's
+  // group was not successfully fetched before they created a tree. This
+  // generally shouldn't happen.
+  group_location_id: number | null;
+  // Lineages selected for tree creation.
+  selected_lineages: string;
```

Comment:
For things that are specifically JSON strings, I prefer to use `JsonString` as the type, and then also have the comment document what should be inside the JSON. I assume this is a JSON array of strings, where each string is a lineage?

Reply:
- [ ] Convert type to `JsonString`

Reply:
The bot's not here yet, but now I'm tempted to hook it up before I get back to this one 😅

Reply:
@vincent-czi this is actually a single string created from the list of lineages.  Setting the type to `selected_lineages: string[];` causes a type error 
![Screen Shot 2022-10-17 at 2 57 05 PM](https://user-images.githubusercontent.com/109251328/196291514-7a2d01cd-a259-4837-9c48-7e28a4f387f2.png)

I actually think it comes from here:
![Screen Shot 2022-10-17 at 2 58 31 PM](https://user-images.githubusercontent.com/109251328/196291719-dcf43d3e-c5ef-418f-8bab-70dc5981cc9f.png)

Does it make more sense to update EventValue to also include `string[]` rather than converting the list to a string?

Reply:
Resolved on a call - using type "JsonString" to indicate to consumers that this is json.  Still using `json.stringify` to change from list to string.